### PR TITLE
Translate faq/gui.po

### PR DESCRIPTION
**Pull request by StevenHsuYL**
Complete translation of faq/gui.po.

Thanks for the review.

HTML preview:
faq-gui

### CODE REVIEW

File: faq/gui.po [hunk 1]
```diff
@@ -4,36 +4,38 @@
...
+msgstr "圖形使用者介面常見問答集"
 
 #: ../../faq/gui.rst:8
 msgid "Contents"
 msgstr "目錄"
 
 #: ../../faq/gui.rst:15
 msgid "General GUI Questions"
-msgstr "常見圖形化使用者界面(GUI) 問題"
+msgstr "一般的圖形使用者介面 (GUI) 問題"
```
Comment (mattwang44) [inline faq/gui.po]:
```suggestion
msgstr "圖形使用者介面 (GUI) 的常見問題"
```

File: faq/gui.po [hunk 2]
```diff
@@ -44,6 +46,12 @@ msgid ""
 "including pointers to the source, see the `Tcl/Tk home page <https://www.tcl."
 "tk>`_.  Tcl/Tk is fully portable to the macOS, Windows, and Unix platforms."
 msgstr ""
+"Python 的標準版本會包含一個 Tcl/Tk 小工具集 (widget set) 的物件導向介面，稱"
+"為 :ref:`tkinter <Tkinter>`。這可能是最容易安裝（因為它已包含在 Python 的大多"
+"數\\ `二進位發行版本 <https://www.python.org/downloads/>`_\\ 中）和使用的。有"
```
Comment (mattwang44) [inline faq/gui.po]:
```suggestion
"數\\ `二進制發行版本 <https://www.python.org/downloads/>`_\\ 中）和使用的。有"
```

File: faq/gui.po [hunk 3]
```diff
@@ -53,28 +61,36 @@ msgid ""
...
+"凍結 (freeze) 是一個能建立獨立應用程式的工具。在凍結 Tkinter 應用程式時，該應"
+"用程式不是真正的獨立，因為該應用程式仍然需要 Tcl 和 Tk 函式庫。"
 
 #: ../../faq/gui.rst:45
 msgid ""
 "One solution is to ship the application with the Tcl and Tk libraries, and "
 "point to them at run-time using the :envvar:`TCL_LIBRARY` and :envvar:"
 "`TK_LIBRARY` environment variables."
 msgstr ""
+"一種解決方法，是將應用程式與 Tcl 和 Tk 函式庫一起提供，並在運行時使用 :"
```
Comment (mattwang44) [inline faq/gui.po]:
```suggestion
"將應用程式與 Tcl 和 Tk 函式庫一併發送是一種解決方法，並在運行時使用 :"
```

File: faq/gui.po [hunk 4]
```diff
@@ -103,17 +125,24 @@ msgid ""
 "function which will be called from the Tk mainloop when I/O is possible on a "
 "file descriptor.  See :ref:`tkinter-file-handlers`."
 msgstr ""
+"在 Windows 以外的平台上，是可以的，你甚至不需要執行緒！但是，你必須稍微調整你"
```
Comment (mattwang44) [inline faq/gui.po]:
```suggestion
"在 Windows 以外的平台上是可以的，你甚至不需要執行緒！但是，你必須稍微調整你"
```

File: faq/gui.po [hunk 5]
```diff
@@ -103,17 +125,24 @@ msgid ""
...
+"的 I/O 程式碼。Tk 具有等效於 Xt 的 :c:func:`XtAddInput()` 的函式呼叫，它能讓"
+"你記錄一個回呼 (callback) 函式，當 I/O 在一個檔案描述符 (descriptor) 上可能可"
+"以進行時，該函式將會從 Tk mainloop 被呼叫。請參閱\\ :ref:`tkinter-file-"
```
Comment (mattwang44) [inline faq/gui.po]:
```suggestion
"你記錄一個回呼 (callback) 函式，當 I/O 在一個檔案描述符 (file descriptor) 上"
"可進行時，該函式將會從 Tk mainloop 被呼叫。請參閱\\ :ref:`tkinter-file-"
```

File: faq/gui.po [hunk 6]
```diff
@@ -103,17 +125,24 @@ msgid ""
...
+"handlers`。"
 
 #: ../../faq/gui.rst:71
 msgid "I can't get key bindings to work in Tkinter: why?"
-msgstr ""
+msgstr "我無法讓鍵連結 (key binding) 在 Tkinter 中作用：為什麼？"
```
Comment (mattwang44) [inline faq/gui.po]:
```suggestion
msgstr "我無法讓鍵繫結 (key binding) 在 Tkinter 中作用：為什麼？"
```

File: faq/gui.po [hunk 7]
```diff
@@ -103,17 +125,24 @@ msgid ""
...
 
 #: ../../faq/gui.rst:73
 msgid ""
 "An often-heard complaint is that event handlers bound to events with the :"
 "meth:`bind` method don't get handled even when the appropriate key is "
 "pressed."
 msgstr ""
+"一個經常聽到的抱怨是，儘管事件處理程式 (event handler) 已經連結到帶有 :meth:"
```
Comment (mattwang44) [inline faq/gui.po]:
```suggestion
"一個經常聽到的抱怨是，儘管事件處理程式 (event handler) 已經繫結到帶有 :meth:"
```

File: faq/gui.po [hunk 8]
```diff
@@ -122,6 +151,9 @@ msgid ""
 "focus command. Usually a widget is given the keyboard focus by clicking in "
 "it (but not for labels; see the takefocus option)."
 msgstr ""
+"最常見的原因是，該連結所應用的小工具並沒有「鍵盤焦點」。請查看 Tk 說明文件中"
+"關於焦點命令的敘述。通常，點擊一個小工具，會讓它得到鍵盤焦點（但不適用於標"
+"籤；請參閱 takefocus 選項）。"
 
 #~ msgid "Qt"
```
Comment (mattwang44) [inline faq/gui.po]: 以下可以刪除了

File: faq/gui.po [hunk 9]
```diff
@@ -122,6 +151,9 @@ msgid ""
 "focus command. Usually a widget is given the keyboard focus by clicking in "
 "it (but not for labels; see the takefocus option)."
 msgstr ""
+"最常見的原因是，該連結所應用的小工具並沒有「鍵盤焦點」。請查看 Tk 說明文件中"
```
Comment (mattwang44) [inline faq/gui.po]:
```suggestion
"最常見的原因是，繫結到的小工具並沒有「鍵盤焦點 (keyboard focus)」。請查看 Tk 說明文件中"
```

**Review by mattwang44 — CHANGES_REQUESTED** [review]

File: faq/gui.po (hunk 8 above)
Comment (StevenHsuYL) [reply faq/gui.po]: All suggestions were applied. 
Thanks for the review! :smile:

**Review by mattwang44 — APPROVED** [review]
LGTM!

**Pull request merged**